stdlib/2/os/__init__.pyi

```diff
@@ -191,16 +191,20 @@ def stat_float_times() -> bool: ...
 def statvfs(path: _PathType) -> _StatVFS: ...  # Unix only
 def symlink(source: _PathType, link_name: _PathType) -> None: ...
 def unlink(path: _PathType) -> None: ...
-def utime(path: _PathType, times: Optional[Tuple[int, int]]) -> None: ...
+def utime(path: _PathType, times: Optional[Tuple[float, float]]) -> None: ...
+
+# TODO onerror: function from OSError to void
 def walk(top: AnyStr, topdown: bool = ..., onerror: Any = ...,
          followlinks: bool = ...) -> Iterator[Tuple[AnyStr, List[AnyStr],
                                                     List[AnyStr]]]: ...
 
-def abort() -> None: ...
-def execl(file: _PathType, *args) -> None: ...
-def execle(file: _PathType, *args) -> None: ...
-def execlp(file: _PathType, *args) -> None: ...
-def execlpe(file: _PathType, *args) -> None: ...
+def abort() -> NoReturn: ...
+def execl(file: _PathType, arg0: Union[bytes, Text], *args: Union[bytes, Text]) -> None: ...
```

Comment:
That was #1077, but I didn't wanna steal their change.  I'll fix these but leave the execv ones alone.

These functions are hard to type. 

I think the arg0 is there because `execl('/bin/echo')` raises a ValueError, because it must be called like this `execl('/bin/echo', '/bin/echo')`

The `execle` and `execlpe` functions are even weirder:
```
def execle(file, *args):
    """execle(file, *args, env)
    Execute the executable file with argument list args and
    environment env, replacing the current process. """
    env = args[-1]
    execve(file, args[:-1], env)
```

but `def execle(file, *args, env)` in python 3 means you have a non-optional keyword arg.  So there's no way to make this perfect.  We could do something like `*args:Union[str, Text, Mapping[str, str]]`  But I think that means you could throw Mapping anywhere in the middle too.